tests/js/spec/views/organizationDashboard/projectCard.spec.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import {ProjectCard} from 'app/views/organizationDashboard/projectCard';
+
+describe('ProjectCard', function() {
+  let wrapper, projectMock;
+  beforeEach(function() {
+    wrapper = mount(
+      <ProjectCard project={TestStubs.Project()} params={{orgId: 'org-slug'}} />,
+      TestStubs.routerContext()
+    );
+
+    projectMock = MockApiClient.addMockResponse({
+      url: '/projects/org-slug/project-slug/',
+      method: 'PUT',
+      data: TestStubs.Project({isBookmarked: false}),
+    });
+  });
+
+  afterEach(function() {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('renders', function() {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('can star', async function() {
+    const star = wrapper.find('Star');
+
+    expect(star.prop('active')).toBe(false);
+    star.simulate('click');
```

Comment:
Can we assert that `star.prop('active')` is true and maybe also test that it can un-bookmark

Reply:
Not if this component is tested in isolation, I can change the tests to mount the whole dashboard instead